### PR TITLE
isnetwork.ISNAutoItStudio version 1.13

### PR DESCRIPTION
**Pull request by ISI-360**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25739)

### CODE REVIEW

File: manifests/i/isnetwork/ISNAutoItStudio/1.13/isnetwork.ISNAutoItStudio.locale.de-DE.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: isnetwork.ISNAutoItStudio
+PackageVersion: 1.13
+PackageLocale: de-DE
+Publisher: isnetwork
+PublisherUrl: https://www.isnetwork.at
+PublisherSupportUrl: https://www.isnetwork.at
```
Comment (OfficialEsco) [inline manifests/i/isnetwork/ISNAutoItStudio/1.13/isnetwork.ISNAutoItStudio.locale.de-DE.yaml:9]: Publisher needs to match the value in `A`dd or `R`emove `P`rograms

```suggestion
Publisher: ISI360
PublisherUrl: https://www.isnetwork.at
PublisherSupportUrl: https://www.isnetwork.at/isn-support/
```

File: manifests/i/isnetwork/ISNAutoItStudio/1.13/isnetwork.ISNAutoItStudio.locale.de-DE.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Publisher: isnetwork
+PublisherUrl: https://www.isnetwork.at
+PublisherSupportUrl: https://www.isnetwork.at
+PrivacyUrl: https://www.isnetwork.at/datenschutz
+Author: ISI360
+PackageName: ISN AutoIt Studio
+PackageUrl: https://www.isnetwork.at
+License: GNU General Public License
+LicenseUrl: https://www.isnetwork.at
+ShortDescription: AutoIt IDE
```
Comment (OfficialEsco) [inline manifests/i/isnetwork/ISNAutoItStudio/1.13/isnetwork.ISNAutoItStudio.locale.de-DE.yaml:16]: Please add a better Description.

```suggestion
ShortDescription: AutoIt IDE
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
A English Locale would also be very nice.

**Comment by ISI-360** [comment]
Uploaded updated yaml files and english locale.

File: manifests/i/isnetwork/ISNAutoItStudio/1.13/isnetwork.ISNAutoItStudio.locale.de-DE.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+PackageName: ISN AutoIt Studio
+PackageUrl: https://www.isnetwork.at
+License: Freeware
+ShortDescription: AutoIt Entwicklungsumgebung
+Description: Entwicklungsumgebung für die Skriptsprache AutoIt
+Tags:
+- AutoIt
+- ISN Studio
+- ISN AutoIt Studio
+- AutoIt IDE
```
Comment (OfficialEsco) [inline manifests/i/isnetwork/ISNAutoItStudio/1.13/isnetwork.ISNAutoItStudio.locale.de-DE.yaml:21]:
```suggestion
- autoit
- isn-studio
- isn-autoit-studio
- autoit-ide
```

File: manifests/i/isnetwork/ISNAutoItStudio/1.13/isnetwork.ISNAutoItStudio.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+PackageName: ISN AutoIt Studio
+PackageUrl: https://www.isnetwork.at
+License: Freeware
+ShortDescription: AutoIt development environment
+Description: Development environment for the scripting language AutoIt
+Tags:
+- AutoIt
+- ISN Studio
+- ISN AutoIt Studio
+- AutoIt IDE
```
Comment (OfficialEsco) [inline manifests/i/isnetwork/ISNAutoItStudio/1.13/isnetwork.ISNAutoItStudio.locale.en-US.yaml:21]:
```suggestion
- autoit
- isn-studio
- isn-autoit-studio
- autoit-ide
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**